src/test/java/redis/clients/jedis/tests/commands/ClientCommandsTest.java

```diff
@@ -228,6 +229,25 @@ public void killUser() {
     jedis.aclDelUser("test_kill");
   }
 
+  @Test
+  public void clientInfo() {
+    String info = client.clientInfo();
+    assertNotNull(info);
+    assertEquals(1, info.split("\n").length);
+    assertTrue(info.contains(clientName));
+  }
+
+  @Test
+  public void clientListWithClientId() {
+    String info = client.clientInfo();
+    Matcher matcher = Pattern.compile("\\bid=(\\d+)\\b").matcher(info);
+    matcher.find();
+    String id = matcher.group(1);
```

Comment:
Is this code on purpose? Or, just to get client id? `clientId()` can be used to get that.